app/views/about/show.html.haml

```diff
@@ -29,7 +29,9 @@
       = render 'registration'
     - else
       .closed-registrations-message
-        - if @instance_presenter.closed_registrations_message.blank?
+        - if Rails.configuration.x.use_ldap
+          %p= t('about.ldap_registrations')
+        - elsif @instance_presenter.closed_registrations_message.blank?
```

Comment:
that makes sense, but if ldap_only is false, then how would you prevent people from signing up with their email addresses?